src/commands/server/start.ts

```diff
@@ -122,11 +122,11 @@ export default class Start extends Command {
       default: DEFAULT_CHE_OPERATOR_IMAGE
     }),
     'che-operator-cr-yaml': string({
-      description: 'Path to a yaml file that defines a CheCluster used by the operator. This parameter is used only when the installer is the operator.',
+      description: `Path to a yaml file that defines a CheCluster used by the operator. This parameter is used only when the installer is the 'operator' or the 'olm'.`,
```

Comment:
Use single quotes, because CI fail. Internal single quotes can be escaped `\'`

Reply:
Sure I can change it ;-) I just saw the backtics on some other parameter description... How come it was accepted there? It's acceptable for multiline strings?

Reply:
Done.